xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -564,7 +794,10 @@ void CPVRTimerInfoTag::GetNotificationText(std::string &strText) const
       strText = StringUtils::Format("%s: '%s'", g_localizeStrings.Get(19224).c_str(), m_strTitle.c_str());
     break;
   case PVR_TIMER_STATE_SCHEDULED:
-    strText = StringUtils::Format("%s: '%s'", g_localizeStrings.Get(19225).c_str(), m_strTitle.c_str());
+    if (IsRepeating())
+      strText = StringUtils::Format("%s: '%s'", g_localizeStrings.Get(826).c_str(), m_strTitle.c_str());
```

Comment:
please add comments describing what the ids are translated to in english for newly added code. I can't ask you to update the existing code, but for now code: please do this so devs can't grep for strings they see on their screens easily and don't have to look up the translation to see what they're writing.